bazel_tools/dev_env_tool/dev_env_tool.bzl

```diff
@@ -4,31 +4,48 @@
 load("@bazel_tools//tools/cpp:lib_cc_configure.bzl", "get_cpu_value")
 load("@rules_sh//sh:posix.bzl", "posix")
 
-def _create_build_content(rule_name, tools, win_paths, nix_paths):
+def _create_build_content(rule_name, is_windows, tools, tool_dependencies, win_paths, nix_paths):
     content = """
 # DO NOT EDIT: automatically generated BUILD file for dev_env_tool.bzl: {rule_name}
+
 package(default_visibility = ["//visibility:public"])
 
 filegroup(
     name = "all",
     srcs = glob(["**"]),
 )
-        """.format(rule_name = rule_name)
+""".format(rule_name = rule_name)
```

Comment:
I don’t understand all those changes. `bazel run @postgresql_dev_env//:pg_ctl -- --help` seems to work just fine for me locally? Is this only about Windows? If so, we don’t actually seem to be using this on Windows.

Reply:
That's odd. If I run that command on `master`, I get:

```
$ bazel run @postgresql_dev_env//:pg_ctl -- --help
INFO: Invocation ID: d52bb82b-794b-4612-8d97-2cb1ffc196ab
ERROR: Cannot run target @postgresql_dev_env//:pg_ctl: Not executable
```

Can you please double-check you were on the `master` branch?

Reply:
Oh you’re right, I ran `bazel run @postgresql_nix//:bin/pg_ctl` which does work but I guess fixing dev-env is sensible even if we do not use this on Windows atm.

Reply:
Planning on making it work on Windows in this PR too; just haven't got around to it yet.